Makefile

```diff
@@ -1,4 +1,4 @@
-SERVERLESS_VERSION = 1.67.2
+SERVERLESS_VERSION = 1.68.0
```

Comment:
1.70.1 in the makefile as well... Seems like there is codes duplication in Make and Docker files.